tests/utils/verification/volumeproperties.go

```diff
@@ -81,7 +81,7 @@ func CheckVolumeListAvailability(hostName string, reqVolList []string) bool {
 	for _, name := range reqVolList {
 		name = strings.Replace(name, "'", "", -1)
 		name = strings.Replace(name, "\"", "", -1)
-		if strings.Contains(volumes, name) != true {
+		if strings.Contains(strings.ToLower(volumes), strings.ToLower(name)) != true {
```

Comment:
So even if we create a volume with a datastore eg. vol@vsanDatastore,
it is listed in docker cli volume list as vol@vsandatastore

We need to document this from the skylight's feature's point of view.

Reply:
Agreed. Will document this difference.